sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -225,15 +282,27 @@ def get_queue_sender(self, queue_name, **kwargs):
         self._handlers.add(handler)
         return handler
 
-    def get_queue_receiver(self, queue_name, **kwargs):
-        # type: (str, Any) -> ServiceBusReceiver
+    def get_queue_receiver(
+        self,
+        queue_name: str,
+        *,
+        session_id: Optional[Union[str, ServiceBusSessionFilter]] = None,
```

Comment:
`ServiceBusSessionFilter` is not part of the public API surface

what mypy error are you getting with this: session_id: Union[str, NEXT_AVAILABLE_SESSION] = None? is there a link to the pipeline log?

Reply:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1277503&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=e3f8f308-6e5f-58aa-3a2a-fe425958de32&l=70

Reply:
removing for now. BUT NEED TO REVISIT!!